db/models.py

```diff
@@ -1 +1,10 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    genre = models.CharField(max_length=255)
```

Comment:
The field name for the `Genre` model should be `name`, not `genre`, as specified in the task description: 'char field `name`, name of the genre with the maximum length of 255 characters.'